backend/src/services/course-module/course-module.service.ts

```diff
@@ -59,17 +59,15 @@ class CourseModule {
       });
     }
 
-    await Promise.all(
-      courseData.map((courseModule) => {
-        return this.create({
-          ...courseModule,
-          description: courseModule.description
-            ? sanitizeHTML(courseModule.description)
-            : null,
-          courseId: dbCourseId,
-        });
-      }),
-    );
+    for (const courseModule of courseData) {
+      await this.create({
+        ...courseModule,
+        description: courseModule.description
+          ? sanitizeHTML(courseModule.description)
+          : null,
+        courseId: dbCourseId,
+      });
+    }
```

Comment:
Need coaches' advice here. I think this block of code can be slow, especially if we're adding a course with hundreds of modules. What if we add another column to the courseModule db, named smth like `order` and then we order by this row, when needed and use `await Promise.all`, as before :thinking:

Reply:
I thought the main purpose of this changes was about fixing the fetching mismatch (in fetching lectures as modules) - about not to save hundreds of lectures but actually a modules (which amount should be much less) 🤔
The idea of ordering modules from our side is nice, but I am not sure if it'd help (like what if they're initially not ordered by fetching) and not sure that for now it would be prior 😕

Reply:
@andanfe @what1s1ove @PavelSukhinin @v0ldemar01 @Liza-Veis

Reply:
Missed the part regarding not saving lectures, in that case I agree, that probably we won't have hundreds of modules, so it shouldn't take a lot of time.
Regarding the order, if we get not ordered data by fetching, I don't think we'll be able to order them from our side at all, so the only way can be to save not ordered data.
Thanks for the answer, just wanted to discuss this point :)

Reply:
Point about not saving lectures is actually the reason of this [ticket](https://trello.com/c/auV50XUH/591-fetch-only-udemy-sections-as-modules) 😅

Reply:
Yeah, firstly I thought it's about ordering the modules, because we had such ticket 😅